apps/application-system/form/src/main.tsx

```diff
@@ -13,15 +13,18 @@ import './auth'
 
 import { environment } from './environments'
 import App from './app/App'
+import { userMonitoring } from '@island.is/user-monitoring'
 
 const activeEnvironment = getActiveEnvironment()
 
-Sentry.init({
-  dsn: environment.sentry.dsn,
-  integrations: [new Integrations.BrowserTracing()],
-  enabled: !isRunningOnEnvironment('local'),
-  environment: activeEnvironment,
-  tracesSampleRate: 0.01,
-})
+if (!isRunningOnEnvironment('local')) {
+  userMonitoring.initDdRum({
+    service: 'application-system-form',
+    applicationId: environment.DD_RUM_APPLICATION_ID,
+    clientToken: environment.DD_RUM_CLIENT_TOKEN,
```

Comment:
But then it doesn't matter since this is client-side rendered 🙈

Reply:
yes, it is all public 👍🏻